api/python/tests/test_bucket.py

```diff
@@ -0,0 +1,56 @@
+import botocore.session
+from botocore.stub import Stubber, ANY
```

Comment:
`ANY` isn't used? also: run pylint

Reply:
I think it makes sense to add something like `test_bucket_io` which tests, e.g. a `put`/`fetch` pair. I realize that `Stubber` isn't _that_ useful because we have to specify all of the responses but it seems strange not to test any of the IO methods.